styles/linter-plus.less

```diff
@@ -15,6 +15,9 @@ linter-message {
   overflow-y: scroll;
   max-height: 150px;
   padding: 4px 8px;
+  &[hidden]{
```

Comment:
This is the native styling for the `hidden` attribute; you shouldn't need to add this.